docs/extend/index.md

```diff
@@ -46,6 +46,8 @@ operation, such as creating a volume.
 In the following example, you install the `sshfs` plugin, verify that it is
 enabled, and use it to create a volume.
 
+> **Note**: This example is intended for instructional purposes only. Once the volume is created, your SSH password to the remote host will be exposed in the volume as plaintext. You should delete the volume as soon as you are done with the example.
```

Comment:
Maybe "exposed as plaintext when inspecting the volume" here, since "in the volume" sounds to me like a container that attaches to the volume can see the password as a file or something.